serverless.yml

```diff
@@ -110,6 +110,13 @@ functions:
           path: clusters/get-pem
           method: get
           private: true
+  remove_pem:
```

Comment:
I feel like a test should be added for adding and removing pems.

Reply:
Sounds good!! Now that the Moto framework in place should be no problem